mbuild/coordinate_transform.py

```diff
@@ -565,13 +603,10 @@ def spin(compound, theta, around):
         The axis about which to spin the compound.
 
     """
-    around = np.asarray(around).reshape(3)
-    if np.array_equal(around, np.zeros(3)):
-        raise ValueError('Cannot spin around a zero vector')
-    center_pos = compound.center
-    translate(compound, -center_pos)
-    rotate(compound, theta, around)
-    translate(compound, center_pos)
+    raise DeprecationError(
+        "spin() has been deprecated. Please use"
+        "Compound.spin(). Deprecated since mBuild version 0.7.0"
```

Comment:
Maybe it would be good to make this message an fstring or something that can be done in a programmatic fashion? But im not sure if there is an easy way to do that for these different methods

Reply:
Thanks for the suggestion.  I added a `__str__` method to `DeprecationError` so the same message isn't repeated throughout the file.